aiohttp/web.py

```diff
@@ -287,3 +287,37 @@ def __call__(self):
 
     def __repr__(self):
         return "<Application>"
+
+
+def run_app(app, *, host='0.0.0.0', port=None,
+            shutdown_timeout=60.0, ssl_context=None):
+    """Run an app locally"""
+    if port is None:
+        if not ssl_context:
+            port = 8080
+        else:
+            port = 8443
+
+    loop = app.loop
+
+    handler = app.make_handler()
+    srv = loop.run_until_complete(loop.create_server(handler, host, port,
+                                                     ssl=ssl_context))
+
+    scheme = 'https' if ssl_context else 'http'
+    prompt = '127.0.0.1' if host == '0.0.0.0' else host
```

Comment:
As I already said -- I read diff from bottom to top) thought that you replaced '0.0.0.0' with localhost and bind to it)